<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>3000</width>
    <height>2000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evolution</string>
  </property>
  <widget class="QPushButton" name="simulate_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="RoundGroup">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cycle Tims</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="loop_count_label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTabWidget" name="plot_tabs">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>2500</width>
     <height>1600</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Number of Population</string>
    </attribute>
    <widget class="PlotWidget" name="plot_animals" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>2400</width>
       <height>1361</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Population Change Data</string>
    </attribute>
    <widget class="PlotWidget" name="plot_animals_change" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>2400</width>
       <height>1361</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Tiger Gene</string>
    </attribute>
    <widget class="PlotWidget" name="plot_tiger_gene" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>2400</width>
       <height>1361</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Wolf Gene</string>
    </attribute>
    <widget class="PlotWidget" name="plot_wolf_gene" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>2400</width>
       <height>1361</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Sheep Gene</string>
    </attribute>
    <widget class="PlotWidget" name="plot_sheep_gene" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>2400</width>
       <height>1361</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>Customise</string>
    </attribute>
    <widget class="PlotWidget" name="plot_custom" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>1111</width>
       <height>531</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>132</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>物种类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="life_type_cbox">
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>132</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>曲线类型</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="curve_type_cbox"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>起始时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="start_age_edit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_9">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>结束时间</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="stop_age_edit"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="plot_custom_btn">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>绘制</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="GrassGroup">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Plant</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="grass_num_edit">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="SheepGroup">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sheep</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="sheep_num_edit">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>80</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="WolfGroup">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Wolf</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="wolf_num_edit">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>60</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="TigerGroup">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tiger</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="tiger_num_edit">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>40</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pause_btn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_10">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>166</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>区间大小</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="age_range_edit">
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
